### DISCUSSION TITLE
Clicking Variable Inspectors did not respond

### DISCUSSION THREAD

**Discussion by wilikeit** [pyscripter/pyscripter#1334] [Q&A]
’https://pyscripter.blogspot.com/2023/10/feature-preview-debug-inspectors.html
The pictures in the blog cannot be displayed, I didn't quite understand how to use inspectors. The blog mentioned that "The Variables window got a new column highlighted in red above", but I do not see this column when I debugging, I click to enlarge the image did not respond.

Is there a problem with my test data？
```

import fun1
def demo():
    '''
    return None
    '''
    fun1.abc()
    b=1
    c=b
    d=c
    e=d
    matrix = [
        [1, 2, 3],
        [4, 5, 6],
        [7, 8, 9]
    ]

    # 打印整个二维列表
    print(matrix)
    f=e

demo()

```

**Comment by pyscripter** [comment]
Variable inspectors support specific classes.  Currently PyScripter ships with just one supporting pandas dataframes.   You can write your own for other data types.

See https://pyscripter.blogspot.com/2023/10/feature-preview-debug-inspectors.html for details.
